### DISCUSSION TITLE
Absolute paths not working when deploying

### DISCUSSION THREAD

**Discussion by jmarroquin-aw** [vercel/vercel#5519] [General]
We're using absolute paths in our next.js project using paths in our tsconfig.json:

    "compilerOptions": {
        "baseUrl": ".",
        "paths": {
            "@/comp/*": [
            "src/components/*"
          ],
            "@/store/*": [
            "store/*"
          ],
            "@/def/*": [
            "src/definitions/*"
          ],`

The problems is that when deploying in vercel, absolute paths are not working:

    09:24:08.041 | Failed to compile.
    09:24:08.041 | ./src/components/car-info/index.tsx:4:27
    09:24:08.041 | Type error: Cannot find module '@/store/reducers' or its corresponding type declarations.

Any ideas how to fix this? 
Do I need to use a different baseUrl when deploying?

**Comment by mcsdevv** [comment]
Can you share the import statement being used along with any custom babel/webpack configurations if present?
